cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -146,7 +146,7 @@ endif
 $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/runsim.do $(COCOTB_LIBS) $(INT_LIBS)
 	set -o pipefail; cd $(SIM_BUILD) && $(LIB_LOAD) MODULE=$(MODULE) TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) COCOTB_SIM=1 \
 	GPI_EXTRA=$(GPI_EXTRA) TOPLEVEL_LANG=$(TOPLEVEL_LANG) PYTHONPATH=$(LIB_DIR):$(PWD):$(NEW_PYTHONPATH) \
-	$(CMD) $(PLUSARGS) -do runsim.do 2>&1 | tee sim.log
+	$(CMD) $(VSIM_ARGS) -do runsim.do 2>&1 | tee sim.log
```

Comment:
```suggestion
	$(CMD) $(RUN_ARGS) -do runsim.do 2>&1 | tee sim.log
```

This is an alternative if SIM_ARGS is not right for here. If not SIM_ARGS, not of the other variables make sense.

Reply:
This is going to be OK, please update docs as well.